apps/web/.env.local.example

```diff
@@ -23,4 +23,8 @@ DATADOG_CLIENT_ID=
 NEYNAR_API_KEY=
 
 FARCASTER_DEVELOPER_FID=
-FARCASTER_DEVELOPER_MNEMONIC=
\ No newline at end of file
+FARCASTER_DEVELOPER_MNEMONIC=
+
+ETHERSCAN_API_KEY=
+BASESCAN_API_KEY=
+TALENT_PROTOCOL_API_KEY=
```

Comment:
Do these keys need to be added to config service or are they already in there?

Reply:
these still need to be added via codeflow 👍